data/schema-config.yaml

```diff
@@ -41,19 +41,22 @@ sequence variant:
         - rsid
   accessible_via:
     name: variants
-    description: 'Retrieve variants data. Example: chr = chr1'
-    filter_by: _id, rsid
-    return: _id, chr, pos, rsid, ref, alt, qual, filter, format
+    description: 'Retrieve variant data. For example: region = chr1:1157520-1158189 or funseq_description = noncoding or rsid = rs2045642915'
```

Comment:
is `funseq_description` one of the properties? I don't see it in properties.

Reply:
yes, it's part of the `annotations` object. Not all variants have this but we still need to allow search